src/main/scala/dx/core/languages/wdl/Block.scala

```diff
@@ -297,12 +297,6 @@ object Block {
         case TAT.ExprPlaceholderSep(sep: TAT.Expr, value: TAT.Expr, _, _) =>
           inner(sep) ++ inner(value)
 
-        // operators on one argument
```

Comment:
We don't need these operators any more?

Reply:
nope - operators were removed in wdlTools AST. Instead, they are converted to function calls.

Reply:
Got it!